src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/CliSyntax.java

```diff
@@ -19,4 +19,15 @@
      * Prevents instantiation of this class.
      */
     private CliSyntax() {}
+
+    /**
+     * Retrieves a list containing all prefixes.
+     *
+     * @return List of all prefixes
```

Comment:
```suggestion
     * @return List of all prefixes.
```

Reply:
Full stop is not required in this case.

Reply:
Please see https://se-education.org/guides/conventions/java/index.html#comments.